### PR TITLE
Added description of wall time checkpoints to newsletter

### PR DESCRIPTION
**Pull request by pbehne**
Ref #26588

### CODE REVIEW

File: modules/doc/content/newsletter/2024/2024_03.md [hunk 1]
```diff
@@ -34,3 +34,10 @@ for a complete description of all MOOSE changes.
 ## PETSc-level Changes
 
 ## Bug Fixes and Minor Enhancements
+
+- Added automatic checkpoints based on simulation wall time. If users
+  forget to turn on checkpoints, a checkpoint will automatically be
+  output every 10 minutes of wall time, or when the current time step
```
Comment (cticenhour) [inline modules/doc/content/newsletter/2024/2024_03.md:40]:
```suggestion
  created every 10 minutes of wall time or when the current time step
```

**Review by cticenhour — CHANGES_REQUESTED** [review]
Minor suggested adjustment noted below and one larger one I am noting here. This is an impactful enough change that it should be highlighted in the main MOOSE changes section as it's own subsection. Developers might miss it being in the bugfixes and minor changes list.

**Comment by loganharbour** [comment]
@pbehne was sitting next to me sans computer; asked for an apply

**Comment by cticenhour** [comment]
There is a part of this review still remaining. In my opinion, this should be moved to it's own section so that it's highlighted a bit more.

**Review by loganharbour — APPROVED** [review]

**Review by cticenhour — APPROVED** [review]

**Pull request merged**